src/site/_includes/js/test/eligibility-programs.test.js

```diff
@@ -2146,4 +2147,31 @@ describe('Program eligibility', () => {
         .is([elig.cnst.wic.CHILD_EXIT_AGE - 1, 99]);
     });
   });
+  describe('Clipper START Program', () => { // Broken!
+    test('Not eligible with default input', () => {
+      expect(elig.clipperStartResult(input).eligible).not.toBe(true);
+    });
+
+    test('Cannot have RTC Clipper Card', () => {
+      input.existingRtcClipperMe = false;
```

Comment:
This does no harm being here (and perhaps adds some readability), though `input.existingRtcClipperMe` is already initialized false thanks to line 595. So, up to you.

Reply:
Changed in [f3038e7](https://github.com/theunitedeffort/theunitedeffort.org/pull/610/commits/f3038e71984f7e2cbbb53b7d59b5ede13d9315f8)